release-please-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "release-type": "go",
+    "bump-minor-pre-major": true,
+    "bump-patch-for-minor-pre-major": true,
+    "include-component-in-tag": false,
+    "packages": {
+        ".": {
+          "package-name": "github.com/microsoftgraph/msgraph-sdk-go",
```

Comment:
this is missing https://github.com/microsoftgraph/msgraph-sdk-go/blob/6c11aca57670ac9e856ab867ba5a4eea9f5b419d/graph_request_adapter.go#L13

Reply:
included the change in this PR